resources/nftables.rb

```diff
@@ -1,9 +1,7 @@
 unified_mode true
 
-action_class do
-  include FirewallCookbook::Helpers
-  include FirewallCookbook::Helpers::Nftables
-end
+include FirewallCookbook::Helpers
+include FirewallCookbook::Helpers::Nftables
```

Comment:
@Stromweld @bmhughes I was tinkering with creating property, but ran into undefined methods messages until I removed the includes from the action_class to their own lines. I'm not sure if my change is the proper way to go about this and would love any insight you might have. I've been reviewing chef.io docs, but haven't found a specific answer yet.

Reply:
That makes sense as action class would only include it for the different action sections. Moving it out should make it available to the properties and actions I believe.

Reply:
Looks like they were `include`'d before but not actually used in the action class, if they were then you'd need both but for the time we an leave it like this.